app/src/main/res/values/dimens.xml

```diff
@@ -15,6 +15,7 @@
   <dimen name="divider_margin_top">8dp</dimen>
   <dimen name="divider_margin_bottom">8dp</dimen>
   <dimen name="bottom_white_space">160dp</dimen>
+  <dimen name="touch_target_size">48dp</dimen>
```

Comment:
Should be ``min_touch_target_size``. Can you also file an issue to update all other clickable views in the app that specify a min height/width to use this dimension? I believe we hardcode 48dp elsewhere, too, and it would be nice to normalize this since it's a really nice idea.

Reply:
I'll rename the dimen and color resource in the next commit. I'll file the issue once this PR is resolved